Snowflake.Data/Core/SFSession.cs

```
@@ -207,10 +207,10 @@ internal void close()
                 authorizationToken = string.Format(SF_AUTHORIZATION_SNOWFLAKE_FMT, sessionToken)
             };
           
-            var response = restRequester.Post<NullDataResponse>(closeSessionRequest);
+            var response = restRequester.Post<CloseResponse>(closeSessionRequest);
             if (!response.success)
             {
-                logger.Warn($"Failed to delete session, error ignored. Code: {response.code} Message: {response.message}");
+                logger.Debug($"Failed to delete session, error ignored. Code: {response.code} Message: {response.message}");

```

All errors are ignored.